packages/connect-form/src/components/advanced-options-tabs/authentication-tab/authentication-gssapi.tsx

```diff
@@ -13,6 +13,11 @@ import {
   parseAuthMechanismProperties,
 } from '../../../utils/connection-string-helpers';
 
+export const GSSAPI_PRINCIPAL_NAME_LABEL = 'Principal';
+export const GSSAPI_SERVICE_NAME_LABEL = 'Service Name';
+export const GSSAPI_CANONICALIZE_HOST_NAME_LABEL = 'Canonicalize Host Name';
+export const GSSAPI_SERVICE_REALM_LABEL = 'Service Realm';
```

Comment:
This might be a totally unnecessary thing I do, but with things like label names I like to not export them for tests if possible and instead make them hardcoded strings. It does mean that every time a label is updated we need to update the test, but I think that helps make the test a bit more solid and regression safe if the label accidentally has a typo added or something.

Reply:
that's a good point, i'll revert everything in the next one :)